common/src/main/java/io/pravega/example/tensorflow/ImageUtil.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.tensorflow;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+import javax.imageio.ImageIO;
+import java.awt.*;
+import java.awt.image.BufferedImage;
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+
+/**
+ * Util class for image processing.
+ */
+public class ImageUtil {
+    private final static Logger LOGGER = LoggerFactory.getLogger(ImageUtil.class);
+    private static ImageUtil imageUtil;
+
+
+    private ImageUtil() {
+
+    }
+
+    /**
+     * It returns the singleton instance of this class.
+     * @return ImageUtil instance
+     */
+    public static ImageUtil getInstance() {
+        if (imageUtil == null) {
+            imageUtil = new ImageUtil();
+        }
+
+        return imageUtil;
+    }
+
+    /**
+     * Label image with classes and predictions given by the ThensorFLow
+     * @param image buffered image to label
+     * @param recognitions list of recognized objects
+     * @return location of the labeled image
+     */
+    public byte[] labelImage(final byte[] image, final List<Recognition> recognitions) {
+        byte[] bytes = null;
+        BufferedImage bufferedImage = imageUtil.createImageFromBytes(image);
+        float scaleX = (float) bufferedImage.getWidth() / (float) 416;
+        float scaleY = (float) bufferedImage.getHeight() / (float) 416;
+        Graphics2D graphics = (Graphics2D) bufferedImage.getGraphics();
+        graphics.setColor(Color.green);
+
+        for (Recognition recognition: recognitions) {
+            BoxPosition box = recognition.getScaledLocation(scaleX, scaleY);
+            //draw text
+            graphics.drawString(recognition.getTitle() + " " + recognition.getConfidence(), box.getLeft(), box.getTop() - 7);
+            // draw bounding box
+            graphics.drawRect(box.getLeftInt(),box.getTopInt(), box.getWidthInt(), box.getHeightInt());
+        }
+
+        graphics.dispose();
+
+        try
+        {
+            ByteArrayOutputStream baos = new ByteArrayOutputStream();
+            ImageIO.write(bufferedImage, "jpg", baos);
+            bytes = baos.toByteArray();
+        }
+        catch (IOException e)
+        {
+            throw  new RuntimeException(e);
+        }
+
+
+        return bytes;
+    }
+
+    /**
+     *
+     * @param image to save
+     * @param target to save image
+     * @return location of the labeled image
```

Comment:
This does not describe what is returned. Change to `JPEG image in a byte array`.
Also note that `target` should be a file name.